P5/Source/Specs/biblStruct.xml

```diff
@@ -68,6 +68,39 @@
       </biblStruct>
     </egXML>
   </exemplum>
+  <exemplum versionDate="2023-02-20" xml:lang="en">
+    <egXML xmlns="http://www.tei-c.org/ns/Examples">
+      <biblStruct type="newspaper">
+        <analytic>
+          <author>
+            <forename>David</forename>
+            <surname>Barstow</surname>
+          </author>
+          <author>
+            <forename>Susanne</forename>
+            <surname>Craig</surname>
+          </author>
+          <author>
+            <forename>Russ</forename>
+            <surname>Buettner</surname>
+          </author>
+          <title type="main">Trump Took Part in Suspect Schemes to Evade Tax Bills</title>
+          <title type="sub">Behind the Myth of a Self-Made Billionaire, a Vast Inheritance From His Father.</title>
```

Comment:
Should the terminal period here be deleted? (I tend to think that titles should not have concluding punctuation in a biblStruct, but I can't find the print edition to see if there's actually a period at the end of the subtitle

Reply:
Thanks, @joeytakeda! Well spotted. I missed that. @sydb kindly took pictures of the print article for me and the subtitle does not have concluding punctuation. I will update that now.